### PR TITLE
sci-mathematics/cgal: version bump to 5.1

### PR DESCRIPTION
**Pull request by epsilon-0**
new maintainer
also build shared libraries

Package-Manager: Portage-3.0.8, Repoman-3.0.1
Signed-off-by: Aisha Tammy <gentoo@aisha.cc>

### CODE REVIEW

File: sci-mathematics/cgal/cgal-5.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,78 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake flag-o-matic
+CMAKE_BUILD_TYPE=Release
```
Comment (DarthGandalf) [inline sci-mathematics/cgal/cgal-5.1.ebuild:7]: Why?

File: sci-mathematics/cgal/cgal-5.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+LICENSE="LGPL-3 GPL-3 Boost-1.0"
+SLOT="0/14"
+KEYWORDS="~amd64 ~arm64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="doc examples +gmp mpfi ntl qt5"
+
+RDEPEND="
+	dev-cpp/eigen
+	dev-libs/boost:=[threads]
+	dev-libs/mpfr:0=
+	sys-libs/zlib:=
```
Comment (DarthGandalf) [inline sci-mathematics/cgal/cgal-5.1.ebuild:28]: zlib should be without :=

https://wiki.gentoo.org/wiki/Proxied_Maintainer_FAQ#Using_a_sub-slot_binder.2C_:.3D

File: sci-mathematics/cgal/cgal-5.1.ebuild (hunk 1 above)
Comment (epsilon-0) [reply sci-mathematics/cgal/cgal-5.1.ebuild:7]: Oof, I thought I had removed `flag-o-matic`
done.

File: sci-mathematics/cgal/cgal-5.1.ebuild (hunk 2 above)
Comment (epsilon-0) [reply sci-mathematics/cgal/cgal-5.1.ebuild]: removed
done

File: sci-mathematics/cgal/cgal-5.1.ebuild (hunk 1 above)
Comment (DarthGandalf) [reply sci-mathematics/cgal/cgal-5.1.ebuild:7]: That was not what I meant, and that's not the line where I left this comment. What's wrong with the default build type of "Gentoo"?

Comment (DarthGandalf) [reply sci-mathematics/cgal/cgal-5.1.ebuild:7]: Then patch that check away :)

Comment (DarthGandalf) [reply sci-mathematics/cgal/cgal-5.1.ebuild:7]: …and send it upstream

Comment (epsilon-0) [reply sci-mathematics/cgal/cgal-5.1.ebuild]: done.

Comment (DarthGandalf) [reply sci-mathematics/cgal/cgal-5.1.ebuild]: I think you forgot `git add`

Comment (epsilon-0) [reply sci-mathematics/cgal/cgal-5.1.ebuild]: Hmmm, no it should be there.

Comment (DarthGandalf) [reply sci-mathematics/cgal/cgal-5.1.ebuild]: Ah, it's not a new file? I see.

File: sci-mathematics/cgal/metadata.xml [hunk 3]
```diff
@@ -1,6 +1,10 @@
 <?xml version='1.0' encoding='UTF-8'?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
+  <maintainer type="person">
+    <email>gentoo@aisha.cc</email>
+    <name>Aisha Tammy</name>
+  </maintainer>
```
Comment (SoapGentoo) [inline sci-mathematics/cgal/metadata.xml:7]: do this in another commit, and add yourself after `sci-mathematics`, and don't forget to add proxy-maint

**Review by SoapGentoo — CHANGES_REQUESTED** [review]

File: sci-mathematics/cgal/metadata.xml (hunk 3 above)
Comment (epsilon-0) [reply sci-mathematics/cgal/metadata.xml:11]: done
